trieste/acquisition/multi_objective/pareto.py

```diff
@@ -74,17 +74,21 @@ def hypervolume_indicator(self, reference: TensorType) -> TensorType:
         return hypervolume_indicator
 
 
-def get_reference_point(front: TensorType) -> TensorType:
+def get_reference_point(
+    observations: TensorType,
+) -> TensorType:
     """
-    reference point calculation method. Note if the front only contains one point, this reference
-    point calculation method will result a reference point the same as the input.
+    default reference point calculation method: a Pareto front will firstly be
```

Comment:
This will do weird things if there are noisy observations?

Reply:
>This will do weird things if there are noisy observations?  

Probably, we already pass the posterior mean-based `observations` hopefully to screen out some observation noise, but whether is the most ideal strategy remains unknown.

Am not immediately aware of an existing dynamic reference updating strategy that is able to take noisy obs into account.    In case of noisy observations, probably an alternative (and published) strategy is just to specify ref point beforehand assuming the knowledge of the rough location of the Pareto frontier (ref footnote 1 of [qNEHVI](https://proceedings.neurips.cc/paper/2021/hash/11704817e347269b7254e744b5e22dac-Abstract.html)), that means a temp strategy that can be used is we do not use this `get_reference_point` in that setting,

Reply:
please can you add some comments saying that the observations input is actuially the posterior mean-based observations. say this gives better perfomance for noisy problems

Reply:
The documentation of `reference_point_spec` has been modified accordingly in the latest [commit](https://github.com/secondmind-labs/trieste/pull/385/commits/3200d3b006bd592b7ec2483a1f8cb7743e125a57). Since the posterior mean-based observation is actually extracted at the acquisition function level, it maybe makes more sense to describe it there for easier usage.  Please see if this makes sense.